lib/mcb/courses_editor.rb

```diff
@@ -34,6 +34,25 @@ def run
       end
     end
 
+    def new_course_wizard
+      %i[title qualifications study_mode accredited_body start_date route maths
```

Comment:
Are these processed in order – we might want to put recruitment_cycle first?

Reply:
Yes, these are currently processed in order.

However I'm not expecting this to be merged before the Rollover branch gets merged in, whereby the recruitment cycle choice step will be rebased out and pulled up to the top level (i.e. specified on the command line, before the interactive part).

Do we want any of the other steps in a different order?